third_party/move/move-compiler-v2/src/options.rs

```diff
@@ -19,13 +19,10 @@ use std::{
 #[derive(Parser, Clone, Debug)]
 #[clap(author, version, about)]
 pub struct Options {
+    #[clap(skip)]
```

Comment:
Do we expose the compiler option to the user? cc @wrwg

Reply:
I don't see a pathway for that.